django/cantusdb_project/main_app/views/views.py

```diff
@@ -255,67 +254,18 @@ def csv_export(request, source_id):
     return response
 
 
-def contact_us(request):
+def contact(request):
     """
-    Function-based view that renders a contact form, access with ``contact``
+    Function-based view that renders the contact page ``contact``
 
     Args:
         request (request): The request
 
     Returns:
-        HttpResponse: Render the contact form
+        HttpResponse: Render the contact page
     """
-    # submitted = False
-    if request.method == "GET":
```

Comment:
@jacobdgm 

I don't understand what these skill_testing_questions are...was the contact form being used for skill testing?

Reply:
It makes sense to me that you wouldn't need this any more, but just checking that there are no dependencies on this functionality.

Reply:
Thanks for checking about this - this is the solution (i.e. fill in the correct letter or the form submit doesn't work) OldCantus found to the problem most other sites solve using Captcha/Recaptcha. I'm confident it's not used elsewhere.